Project SF/Views/NavigationItem.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  NavigationItem.swift
+//  Project SF
+//
+//  Created by Roman Esin on 12.07.2020.
+//
+
+import SwiftUI
+
+struct NavigationItem: View {
+    let imageName: String
+    let action: () -> Void
+
+    var body: some View {
+        Button(action: action, label: {
+            Image(systemName: "plus")
```

Comment:
Don't you need to set this to be the imageName?

Reply:
Oh true, will fix a bit later